packages/inventory-insights/src/Insights.js

```diff
@@ -128,6 +128,12 @@ class InventoryRuleList extends Component {
 
     buildRows = (activeReports, kbaDetails, filters, rows, searchValue = '', kbaLoading = false) => {
         const builtRows = flatten(activeReports.map((value, key) => {
+            const riskToString = {
```

Comment:
@LI0131 while there's not many other places we might be usin this, lets throw this in `constants.js` just cuz its the kinda thing that should be in there <3